Sources/MapboxNavigation/PreviewViewControllerDelegate.swift

```diff
@@ -3,40 +3,82 @@ import CoreLocation
 import MapboxDirections
 import MapboxCoreNavigation
 
-// :nodoc:
+/**
+ `PreviewViewControllerDelegate` allows to observe `Banner` presentation and dismissal events.
+ */
 public protocol PreviewViewControllerDelegate: AnyObject, UnimplementedLogging {
     
+    /**
+     Tells the delegate that the `Banner` is about to be presented on the screen.
+     
+     - parameter previewViewController: `PreviewViewController` instance that performs banner
+     presentation.
+     - parameter banner: `Banner` that will be presented.
+     */
     func previewViewController(_ previewViewController: PreviewViewController,
                                willPresent banner: Banner)
     
+    /**
+     Tells the delegate that the `Banner` was presented on the screen.
+     
+     - parameter previewViewController: `PreviewViewController` instance that performs banner
+     presentation.
+     - parameter banner: `Banner` that was presented.
+     */
     func previewViewController(_ previewViewController: PreviewViewController,
                                didPresent banner: Banner)
     
+    /**
+     Tells the delegate that the `Banner` will disappear from the screen.
+     
+     - parameter previewViewController: `PreviewViewController` instance that performs banner
+     dismissal.
+     - parameter banner: `Banner` that will be dismissed.
+     */
     func previewViewController(_ previewViewController: PreviewViewController,
                                willDismiss banner: Banner)
     
+    /**
+     Tells the delegate that the `Banner` disappeared from the screen.
+     
+     - parameter previewViewController: `PreviewViewController` instance that performs banner
+     dismissal.
+     - parameter banner: `Banner` that was dismissed.
+     */
     func previewViewController(_ previewViewController: PreviewViewController,
                                didDismiss banner: Banner)
 }
 
 // :nodoc:
```

Comment:
Should this `:nodoc:` be deleted?

Reply:
Yep, makes sense. `PreviewViewControllerDelegate` is documented anyway.